opencart 1.5.6.4/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+
+services:
```

Comment:
I would just change the service structure in all of the Docker Compose files a tiny bit.

```
services:
    app:
        ...
    database:
        ...

volumes:
    app:
    database:
```

This would make the Docker Compose generated container and volume names more human-friendly. Instead of `opencart2200_opencart_1` it would be `opencart2200_app_1`.